ocelot/urls.py

```diff
@@ -18,6 +18,7 @@
 from django.urls import path
 
 from leaderboard.views import frontpage
+from leaderboard.views import leaderboardpage
```

Comment:
@snukky: just `leaderboard` would be much nicer/more clean? For `frontpage`, I simply did not have a better name :) For this case, `leaderboard.views.leaderboard` seems like a good enough name. Thoughts?

Reply:
I agree. I thought that it was a naming convention. Changed.